src/deadline/client/ui/dialogs/submit_job_to_deadline_dialog.py

```diff
@@ -82,6 +82,8 @@ def __init__(
         auto_detected_attachments: AssetReferences,
         attachments: AssetReferences,
         on_create_job_bundle_callback,
+        on_pre_submit_callback=None,
+        on_post_submit_callback=None,
```

Comment:
It would be great to add these new arguments to the docstring above, and describe the purpose of the parameters there. E.g. if the purpose is that the callback might modify parameters, add boilerplate to the template, or similar, would be great to mention that.

Reply:
Ah I forgot that thanks!